crates/cli/tests/runner/mod.rs

```diff
@@ -43,12 +43,13 @@ struct StoreContext {
 
 impl StoreContext {
     fn new(input: &[u8], capacity: usize) -> Self {
-        let mut wasi = WasiCtxBuilder::new().inherit_stdio().build();
```

Comment:
I was getting a warning about the `mut` being unnecessary but removing it caused Rust Analyzer to complain that the code had compilation errors even though cargo was able to compile it. I opted to just set the streams in the `WasiCtxBuilder` instead to get around the issue.